README.adoc

```diff
@@ -1,5 +1,6 @@
 :icons: font
 :source-highlighter: prettify
+:doctype: book
```

Comment:
A table of contents for this could also be helpful:
```suggestion
:doctype: book
:toc: left
```

or this and place `toc::[]` after the preamble in `= Microservice Extensibility Concepts`
```suggestion
:doctype: book
:toc: macro
```

`:toclevels: 1` might make it look better if there's too many subheadings